src/views/explorer/components/extrinsicsList/index.tsx

```diff
@@ -66,7 +66,7 @@ export const ExtrinsicsList = polymorphicComponent<'div'>((_props, ref) => {
 
   useEffect(() => {
     const extrinsics = Array.from(blocksData.values())
-      .flatMap((block) => block?.extrinsics ?? [])
```

Comment:
There is one more route `explorer/extrinsics` where want to filter out the first two extrinsics of each block